csharp/Platform.Collections/Lists/IListExtensions.cs

```diff
@@ -258,7 +246,18 @@ public static bool ContentEqualTo<T>(this IList<T> left, IList<T> right)
             }
             return true;
         }
-
+        
+        /// <summary>
+        /// <para>If the transmitted list is not empty, creates an array with and copies all elements of the list into it.</para>
```

Comment:
`Creates an array by copying all elements from the list that satisfy the predicate. If no list is passed, null is returned.`